paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -320,18 +397,19 @@ internal class CustomerSheetViewModel(
     }
 
     private fun onEditPressed() {
-        if (viewState.value.isEditing) {
+        if (!customerState.value.canEdit) {
+            return
+        }
```

Comment:
Is it possible to end up in this state? Usually, the edit button is hid when `canEdit` is false, right?

Reply:
Yeah it won't be possible. This was added as a precaution most likely.